ludwig/encoders/text_encoders.py

```diff
@@ -40,15 +41,22 @@ def __init__(
             pretrained_model_name_or_path
         )
         self.reduce_output = reduce_output
+        self.transformer.trainable = False
```

Comment:
Here and in all other encoders add a `trainable=False` parameter to init, so that users have the option to finetune.